src/plugins/intel_gna/tests/functional/pass_tests/concat_restrictions.cpp

```diff
@@ -77,20 +77,20 @@ struct MatmulConcatAxis {
             break;
         }
 
-        size_t mulConstSize = ngraph::shape_size(mulConstShape);
+        size_t mulConstSize = ov::shape_size(mulConstShape);
         std::vector<float> weights1(mulConstSize);
         std::vector<float> weights2(mulConstSize);
         std::iota(weights1.begin(), weights1.end(), 0.0f);
         std::iota(weights2.begin(), weights2.end(), 0.0f);
         auto constMul1 = ngraph::builder::makeConstant<float>(ngPrc, mulConstShape, weights1);
         auto constMul2 = ngraph::builder::makeConstant<float>(ngPrc, mulConstShape, weights2);
-        auto matmul1 = std::make_shared<ngraph::opset8::MatMul>(params[0], constMul1, false, true);
+        auto matmul1 = std::make_shared<ov::opset10::MatMul>(params[0], constMul1, false, true);
         concatInputs.push_back(matmul1);
-        auto matmul2 = std::make_shared<ngraph::opset8::MatMul>(params[0], constMul2, false, true);
+        auto matmul2 = std::make_shared<ov::opset10::MatMul>(params[0], constMul2, false, true);
         concatInputs.push_back(matmul2);
         auto concat = ngraph::builder::makeConcat(concatInputs, axis);
 
-        ngraph::ResultVector results{std::make_shared<ngraph::opset8::Result>(concat)};
+        ov::ResultVector results{std::make_shared<ov::opset10::Result>(concat)};
```

Comment:
ov::opset11 is available. why not used here?

Reply:
AFAIK opset10 is the newest one to be used.

Reply:
do we keep unused opsets in code then?

Reply:
opset11 won't work here